examples/vision/mixUp.py

```diff
@@ -3,7 +3,7 @@
 Author: [Sayak Paul](https://twitter.com/RisingSayak)
 Date created: 2020/03/06
 Last modified: 2020/03/06
-Description: How to implement mixup to augment images for image classification problems.
+Description: Applying the mixup technique for data augmentation to aid with image classification.
```

Comment:
alternative description suggestion:

```suggestion
Description: Data augmentation using the mixup technique for image classification.
```

WDYT? cc @fchollet